src/test/java/harness/TestUsers.java

```diff
@@ -44,6 +44,14 @@
   // name of the group that includes CLI test users and has spend profile access
   public static final String CLI_TEST_USERS_GROUP_NAME = "cli-test-users";
 
+  // these are the same scopes requested by Terra service swagger pages, plus the cloud platform
+  // scope. test users need the cloud platform scope to talk to GCP directly (e.g. to check the
+  // lifecycle property of a GCS bucket, which is not stored as WSM metadata)
```

Comment:
This makes me think you want a separate test helper user in addition to the test users. That way they'd be the same as in production.

Reply:
Agreed. We should test with the same scopes we give to users.